### DISCUSSION TITLE
I'd like to be part of the development of this app

### DISCUSSION THREAD

**Discussion by TechnoSmack** [microrealestate/microrealestate#140] [Ideas]
I would enjoy aiding the development of this app in any way or shape. please contact me or email me back :)

**Comment by camelaissani** [comment]
Thank you for your interest to this project. Feel free to create PRs to enhance the documentation, to translate the app, to fix bugs or to implement new functionalities.

Please tell me if you need more guidance.

**Comment by camelaissani** [comment]
Another way to help is to sponsor the project 🙂

https://github.com/sponsors/camelaissani
